cas-server-integration-ehcache/src/main/java/org/jasig/cas/ticket/registry/EhCacheTicketRegistry.java

```diff
@@ -50,7 +51,7 @@
  * @author Andrew Tillinghast
  * @since 3.5
  */
-public final class EhCacheTicketRegistry extends AbstractDistributedTicketRegistry implements InitializingBean {
+public final class EhCacheTicketRegistry extends AbstractCrypticTicketRegistry implements InitializingBean {

```

Comment:
Also worthy to point out that the changeset here is 100% backwards compatible, even though per release strategy, it may not have to be.